analysis_module/utils.py

```diff
@@ -265,4 +268,6 @@ def get_ecs_url(request_type: NLPRequest.FeaturesType):
         return urljoin(ENTRYEXTRACTION_ECS_ENDPOINT, "/extract_entries")
     elif request_type == NLPRequest.FeaturesType.GEOLOCATION:
         return urljoin(GEOLOCATION_ECS_ENDPOINT, "/get_geolocations")
+    elif request_type == NLPRequest.FeaturesType.TOPICMODEL:
+        return urljoin(TOPICMODEL_ECS_ENDPOINT, "/get_excerpt_clusters")
     return None
```

Comment:
use dict mapping instead of muliple if conditons 
Reference: https://stackoverflow.com/questions/60792742/using-dict-mapping-instead-of-multiple-if-statements

Reply:
done